uat/testing-features/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -1,15 +1,14 @@
 @LogManager
 Feature: Greengrass V2 LogManager
 
-    As a customer, I want to selectively upload my logs and metrics to AWS Cloudwatch to save cost.
+    As a customer, I can configure the log manager to upload my component logs to cloudwatch
 
     Background:
         Given my device is registered as a Thing
         And my device is running Greengrass
 
-    Scenario: LogManager-1-T1: configure the log manager component using a componentLogsConfiguration list and logs are uploaded to
-    CloudWatch
-        Given I create a log directory for component UserComponentM 
+    Scenario: LogManager-1-T1: component logs are uploaded to cloudwatch
+        Given I create a log directory for component UserComponentM
```

Comment:
Nit: Should we specify the log directory name here (UserComponentMLogDirectory)?

Reply:
The log directory name is randomly created to avoid any sort of collision with another test wanting to use the same name

Reply:
It can still be random. I'm saying more in terms of interpolation <log-directory>. 

Because, down it the log manager component configuration, test authors must know that the log directory created i the test will be component_name + "LogDirectory" in the scenario context. It's an improvement not a blocker for this PR.